tests/loader_layer_tests.cpp

```diff
@@ -2093,7 +2106,11 @@ TEST_F(LayerExtensions, ExplicitBothDeviceExtensions) {
 
     // Make sure only the appropriate function pointers are NULL as well
     ASSERT_NE(nullptr, dev->vkGetDeviceProcAddr(dev.dev, "vkTrimCommandPoolKHR"));
-    ASSERT_NE(nullptr, dev->vkGetDeviceProcAddr(dev.dev, "vkGetSwapchainStatusKHR"));
+
+    PFN_vkGetSwapchainStatusKHR pfnGetSwapchainStatusKHR =
+        reinterpret_cast<PFN_vkGetSwapchainStatusKHR>(dev->vkGetDeviceProcAddr(dev.dev, "vkGetSwapchainStatusKHR"));
+    ASSERT_NE(nullptr, pfnGetSwapchainStatusKHR);
```

Comment:
FYI the `handle_assert` helpers are for this situation (thanks win32...)
```suggestion
    handle_assert_null(pfnGetSwapchainStatusKHR);
```

Reply:
Updated all occurrences in the file.